modules/services/sketchybar/default.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) literalExpression maintainers mdDoc mkEnableOption mkIf mkPackageOptionMD mkOption optionals types;
+
+  cfg = config.services.sketchybar;
+
+  configFile = pkgs.writeScript "sketchybarrc" cfg.config;
+in
+
+{
+
+  meta.maintainers = [
+    maintainers.azuwis or "azuwis"
+  ];
+
+  options.services.sketchybar = {
+    enable = mkEnableOption (mdDoc "sketchybar");
+
+    package = mkPackageOptionMD pkgs "sketchybar" { };
+
+    extraPackages = mkOption {
+      type = types.listOf types.package;
+      default = [ ];
+      description = mdDoc ''
+        Extra packages to add to PATH.
+      '';
+      example = literalExpression "[ pkgs.jq ]";
+    };
+
+    config = mkOption {
+      type = types.lines;
+      default = "";
+      example = literalExpression ''
```

Comment:
`literalExpression` is wrong here, should just be a plain string

Reply:
Done in latest push.